### DISCUSSION TITLE
Impossibile connettermi da remoto

### DISCUSSION THREAD

**Discussion by Cupra1979** [indomus/forum#1535] [HUB: Home Assistant]
Ciao a tutti!

Sto provando da tempo a configurare correttamente l'accesso remoto al mio Raspberry (PI 4 4GB con HA), ma non ci salto fuori :(
Credo di aver eseguito correttamente tutti i passaggi come da guida su Indomus, ma non riesco assolutamente ad accedere.

**Premessa**: uso HA solo per "esporre" verso Homey alcuni sensori o device non supportati nativamente dall'altro HUB, il mio livello su HA è abbastanza "basic".

In dettaglio:
- ho seguito alla lettera le istruzioni di configurazione sul sito (https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-con-distribuzione-hassio/)
- ho aperto la porta 8123 verso l'ip del Raspberry, rendendolo statico (router FritzBox con fibra FTTH di MyNet)

Se provo ad accedere da remoto risulta impossibile e va in timeout...è demoralizzante :(
Tra l'altro, non riesco nemmeno più ad accedere localmente e non so cosa cavolo sia successo...devo provare a collegare un monitor direttamente via HDMI e verificare se HA si è avviato correttamente dopo del riavvio post-configurazione di DuckDNS.

Non so sinceramente che pesci pigliare...grazie mille in anticipo a chiunque saprà darmi una mano!!!!

**Comment by indomus** [comment]
Sei sicuro di avere, tramite  fibra FTTH di MyNet, un IP WAN pubblico?

**Reply by Cupra1979** [reply]
Questo è quanto da contratto (a parte che da qualche mese è diventata senza sovra costi 1000/300:
![image](https://user-images.githubusercontent.com/69384072/155366490-fa94fd75-cb79-4423-92e2-52317567b758.png)

**Reply by indomus** [reply]
Dovrebbe esser garantito, anche se "indirizzo IP statico" vuol dir tutto e niente. Fai dei test: prova a collegarti all'interfaccia del modem da remoto (ovviamente attivala da configurazione, immagino su porta 80), vedi se riesci.

**Reply by Cupra1979** [reply]
Riesco a collegarmi senza problemi.
Ed ho anche risolto il problema della connessione da remoto!! Ho notato che la registrazione su DuckDNS che feci a suo tempo riportava un iP non più aggiornato (prima che lo rendessi fisso per il Rasp).
Finalmente riesco a collegarmi da remoto!!! :)

**Reply by Valeriinat** [reply]
Salve, io ho lo stesso problema, però uso un mini pc con la macchina virtuale. Come posso risolvere?

**Reply by 50494554524F** [reply]
> Salve, io ho lo stesso problema, però uso un mini pc con la macchina virtuale. Come posso risolvere?

con l'addon tor ci vogliono 5 minuti... non serve che ti scriva la procedura, la trovi nella doc dell'addon, molto semplice
